Server/cartemplate.rb

```diff
@@ -0,0 +1,78 @@
+class Car
+	attr_reader :manufacturer, :model, :type, :price
+
+	def initialize(manufacturer, model, type, price)
+		@manufacturer = manufacturer
+		@model = model
+		@type = type
+		@price = price
+	end
+
+	def to_s
+		
```

Comment:
Since your #to_s has no return value, any attempt to `puts car` will just give nil. If you haven't decided what should go in the method, just comment it out.